### DISCUSSION TITLE
Differenciate between working time and billable hours

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#3672] [Questions & Answers]
Hello all,

we've an internal project for working time, holidays and sick leave and several projects where billable hours are maintained  for creating invoices to customers.
So far so good.
But there is a problem weithin reporting.
When I report on working hours for my staff,  all project times  are considered (and not only the working time created on the internal project).
Is there a possibility to mark activities/projects as 'working time relevant' so that omly these get into account when reporting on working time.
Background: billable hours (for different customers at the same time) are not the same as working hours.

Best regards
Klaus

**Comment by kevinpapst** [comment]
Kimai is a project time-tracking software.

While you can mix attendance time and billable hours, Kimai was never made for handling working-time-accounts.
There will only be a problem if you track attendance from start to end and then add billable hours on top.

Simply create multiple entries for a day:
- start to first billable hour
- billable hour 1 
- billable hour 2
- billable hour 3
- break (do not track this one=
- billable hour 4
- internal work until leaving

I assume for this example you have 5 entries currently. One for the entire day (or two for attendance) and 4 for billable hours?
Try to use the setup above instead (do not track break time).

**Reply by idemus** [reply]
Hi Kevin,
I know that Kimai was not made for working time attendance but Kimai is so good and not far away to handle this (in my oppinion) in combination with the holiday and sick leave plugin.
So would it be possible to integrate a flag 'work time' on project and/or activity level  so that those ativities/projects (where 'work time' = false) won't be considered for working time in reporting.
That flag should be customizable (default value in project/activity) and should be controlled by permissions.
Of course I would pay for it.

Best regards
Klaus
